package.json

```
@@ -19,6 +19,7 @@
     "build": "cross-env NODE_OPTIONS='--max-old-space-size=4096' CI=false npm-run-all -s build:css build:js",
     "build:css": "cross-env NODE_ENV=production postcss src/tailwind.css -o src/index.css",
     "build:js": "cross-env NODE_ENV=production react-app-rewired build",
+    "build:config": "ts-node src/test/fixtures/config/configuration-file.tsx",
```

```suggestion
    "build:config": "ts-node src/test/fixtures/config/configuration-file.ts",
```

the rest as ts files as well, to minimise ambiguity on purpose of files

---

done